PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalSoftDropResponse.h

```diff
@@ -89,27 +95,27 @@ class AliAnalysisTaskEmcalSoftDropResponse : public AliAnalysisTaskEmcalJet {
   Double_t                      fBeta;                      ///< Beta (softdrop definition)
   EReclusterizer_t              fReclusterizer;             ///< Reclusterizing algorithm
   Double32_t                    fSampleFraction;            ///< Fraction of statistics used for the analysis
+  Float_t                       fMinFractionShared;         ///< only fill histos for jets if shared fraction larger than X  
+
   Bool_t                        fUseChargedConstituents;    ///< Use charged constituents for softdrop
   Bool_t                        fUseNeutralConstituents;    ///< Use neutral constituents for softdrop
   TString                       fNameMCParticles;           ///< Name of the MC particle container
   TRandom                       *fSampleSplitter;           ///< Sample splitter
   TRandom                       *fSampleTrimmer;            ///< Sample trimmer
   TBinning                      *fPartLevelPtBinning;       ///< Particle level pt binning
   TBinning                      *fDetLevelPtBinning;        ///< Detector level pt binning
-  RooUnfoldResponse             *fZgResponse;               //!<! RooUnfold response object
-  RooUnfoldResponse             *fZgResponseClosure;        //!<! RooUnfold response for the closure test
-  TH2                           *fZgPartLevel;              //!<! Zg vs. pt at particle level
-  TH2                           *fZgDetLevel;               //!<! Zg vs. pt at detector level
-  TH2                           *fZgPartLevelTruncated;     //!<! Zg vs. pt at particle level after truncation at
-  TH2                           *fZgPartLevelClosureNoResp; //!<! Zg vs. pt at particle level for closure test (jets not used for response)
-  TH2                           *fZgDetLevelClosureNoResp;  //!<! Zg vs. pt at detector level for closure test (jets not used for response)
-  TH2                           *fZgPartLevelClosureResp;   //!<! Zg vs. pt at particle level for closure test (jets used for response)
-  TH2                           *fZgDetLevelClosureResp;    //!<! Zg vs. pt at detector level for closure test (jets used for response)
+  Bool_t                        fIsEmbeddedEvent;           ///<true if the event is an embedded event       
+  TString                       fPartLevelJetName;          ///< Name of the particle level jet container  
+  TString                       fDetLevelJetName;          ///< Name of the detector (or hybrid if embedding)  level jet container  
+  TString                       fUSLevelJetName;          ///< Name of the unsubtracted hybrid level jet container
```

Comment:
Also here clearer name (i.e. fUnSubLevelJetContainer) to make its meaning clearer.